packages/loot-design/src/components/common.js

```diff
@@ -593,7 +593,7 @@ export function Menu({ header, footer, items: allItems, onMenuSelect }) {
     el.current.addEventListener('keydown', onKeyDown);
 
     return () => {
-      el.current.removeEventListener('keydown', onKeyDown);
+      el.current && el.current.removeEventListener('keydown', onKeyDown);
```

Comment:
Hm… should we save the `el.current` value used to add the event listener so that this doesn’t leak the listener?

Reply:
Good idea: https://github.com/actualbudget/actual/pull/696/commits/493e2ffe7cb372d27aa652226bb9b2deaa4d22dd